src/parser/generated_parser.ts

```diff
@@ -414,55 +416,55 @@ export class GeneratedParser extends Parser {
         let literal, mark;
         mark = this.mark();
         if ((literal = this.expect("+="))) {
-            return new astnodes.Add();
+            return pegen.augoperator(this, Add);
```

Comment:
augoperator is back operators themselves need to be changed in the grammar still

Reply:
this is done